src/index.ts

```diff
@@ -69,7 +74,10 @@ export function restoreResumableFields(id: string, options?: RestoreOptions): vo
     if (document.dispatchEvent(resumeEvent)) {
       const field = document.getElementById(fieldId)
       if (field && (field instanceof HTMLInputElement || field instanceof HTMLTextAreaElement)) {
-        if (field.value === field.defaultValue) {
+        if (field instanceof HTMLInputElement && (field.type === 'checkbox' || field.type === 'radio')) {
+          field.checked = true
```

Comment:
My understanding here is that in `shouldResumeField`, we check whether the checked state of the field is != the default state: `field.checked !== field.defaultChecked`

Yet this always restores the checked state to `true`. Could there be a case where the default state of a field is checked, the user unchecks it but we restore the state as checked? 🤔

Reply:
Love when it's a matter of a small change :) switching `true` to `!field.defaultChecked` appears to work.